api/src/apps/routes.rs

```diff
@@ -186,14 +190,16 @@ async fn change_status(
 }
 
 #[get(
-    "/<app_name>/logs/<service_name>?<since>&<limit>",
-    format = "text/plain"
+    "/<app_name>/logs/<service_name>?<since>&<limit>&<download>",
```

Comment:
I struggle a bit with the term `download`. What do you think about following suggestion?

```suggestion
    "/<app_name>/logs/<service_name>?<since>&<limit>&<asAttachment>",
```

Reply:
camelCase throws a cargo warning. Perhaps you referred to using `as_attachment`